app/backends/workflow-orchestration.ts

```diff
@@ -526,138 +537,162 @@ export async function handleWorkItemUpdate(
   update: WorkItemUpdate,
   operation: object,
   logger: Logger): Promise<void> {
-  const { workItemID, status, hits, results, scrollID, errorMessage } = update;
+  const { workItemID, hits, results, scrollID } = update;
+  let { errorMessage, status } = update;
+  let createdWorkItem = false;
```

Comment:
Can you change this do `didCreateWorkItem` or something similar that indicates it is a boolean? I was confused where it was assigned and was thinking it was a `WorkItem` that got created.

Reply:
Done.